modules/external_state/external-state.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "external-state"
+linkTitle: "external-state"
+date: 2021-07-21
+draft: false
+weight: 1
+description: Adds the ability to refer to external, preexisting terraform states in opta.
+---
+
+This module grants users the ability to import outputs from 
+[external terraform backends](https://www.terraform.io/language/settings/backends) (e.g. pre-existing
+or created outside of opta, or another Opta service/environment). All the terraform outputs of that state
+will be captured and referenceable under the `outputs` output for this module. 
+
+For instance, you can refer values such as the name of an S3 bucket created in another Opta service like so:
```

Comment:
I can see some possible confusion for some users between the S3 used for the backend and the S3 bucket used in this example. But they are different things.

Can we use another example? may be name of EKS cluster, Database name or VPC id these are usually common.